### PR TITLE
<UPD>: Update how generate emissions pre-generates

### PR DESCRIPTION
**Pull request by SEAJang**
# Pull Request Key Information

## Reason for change

The emissions that are generated for pre-sim start date were not ordered correctly

## What was changed

- Update generate emissions in sources
- Update existing unit tests
- Add more unit tests to ensure the dates are correct.

## Intended Purpose

Update the code such that the emissions are generated based on oldest date first

## Level of version change required

Patch

## Testing Completed

All unit tests pass - old and new
[results.txt](https://github.com/user-attachments/files/16418098/results.txt)

The E2E tests have been updated to reflect the new generate emissions changes. 
[V4_simple_non_repairable_emissions_300e5ec164c2571e1d50afdafe37c6ce2d1ad8a3.log](https://github.com/user-attachments/files/16418156/V4_simple_non_repairable_emissions_300e5ec164c2571e1d50afdafe37c6ce2d1ad8a3.log)
[V4-simple_repairable_emissions_300e5ec164c2571e1d50afdafe37c6ce2d1ad8a3.log](https://github.com/user-attachments/files/16418157/V4-simple_repairable_emissions_300e5ec164c2571e1d50afdafe37c6ce2d1ad8a3.log)

## Target Issue

Put GitHub keywords to link the pull request to the target issue here. For example: Closes #XX.

## Additional Information

Include any other important information here.

### CODE REVIEW

**Comment by ThomasGalesloot** [comment]
Was this a bug or is this a change?

**Comment by SEAJang** [comment]
> Was this a bug or is this a change?

It was a change. Previously the dates for the pre-sim start went from newest to oldest, and then all emissions after that went from oldest to newest. now it's constant oldest to newest.

File: LDAR_Sim/testing/unit_testing/test_initialization/test_sources/test_generate_emissions.py
Comment (ThomasGalesloot) [inline LDAR_Sim/testing/unit_testing/test_initialization/test_sources/test_generate_emissions.py:1]: where are the two new mock sources used?

**Comment by ThomasGalesloot** [comment]
Does either activate emissions or emissions logic around pre-simulation emissions need to be updated as part of this change?

File: LDAR_Sim/testing/unit_testing/test_initialization/test_sources/test_generate_emissions.py
Comment (SEAJang) [reply LDAR_Sim/testing/unit_testing/test_initialization/test_sources/test_generate_emissions.py:1]: test_generate_preemissions and test_generate_singleemissions

**Comment by SEAJang** [comment]
> Does either activate emissions or emissions logic around pre-simulation emissions need to be updated as part of this change?

Nope.

**Review by ThomasGalesloot — APPROVED** [review]

**Pull request merged**